checker/src/org/checkerframework/checker/genericeffects/GenericEffectVisitor.java

```diff
@@ -148,69 +138,958 @@ public Void visitMethod(MethodTree node, Void p) {
         return ret;
     }
 
+    /**
+     * Method that visits variable tree nodes and adds their effects to the stacks set up in the constructor.
+     *
+     * @param node The variable tree node that was encountered during checking.
+     * @param p Void
+     * @return Void
+     */
+    @Override
+    public Void visitVariable(VariableTree node, Void p)
```

Comment:
We shouldn't be doing anything at all for regular local variables.  If this is how instance fields show up, please be explicit about that.  If this is how both show up, then this code will mix up the instance initialization with method initialization.